core-guides/core-data-curators.md

```diff
@@ -0,0 +1,172 @@
+
+1. Follow the [GitHub's issue tracker](https://github.com/datasets/registry/issue) and look for a package with 3-star priority, or
+
+2. Get your own data and start preparing it.
+
+Which one you should go to first is really up to you. The difference, however, is that some are more needed than others and 
+you could use that knowledge and skill to tackle on some of the project's priorities.
+
+Other than that, the workflow and requirements is pretty much the same.
+
+## Following urgencies
+
+The main advantage of working with a previously open issue is the fact you are introduced to some work that is already 
+developed. Chances are that someone has already took care of finding the best source, convert it to CSV or XLSX and then your 
+work is reduced significantly.
+
+## Your Own Taste
+
+You can also search for topics of your preference, or even suggest them in the issues page if they are not there already.
+
+
+# Preparing the Data
+
+Now, back to the workflow. This is the troublesome part. The job is to, first, find the best and most reliable source, convert 
+if to CSV or XLSX and then work on preparing the data package. If you do not know what `CSV` is, we advise you to read the 
+[Data Guides](data-guides/).
+
+## Convert to .CSV
+
+As mentioned before, the first step is to have a source `csv` file and you can do that, for instance, by click in the right 
+tab in the [World Bank of Data](http://data.worldbank.org/). See picture below:
+
+![image]({{ site.url }}/images/export.jpg)
+
+Chances are, many world data references have a similar option.
+
+## Directory structure
+
+In order to keep everything organized and as "universal" as possible, we have found a structure that pleases the most of us 
+that any contributor should also go by:
+
+```
+dir
+   data
+      id-name.csv
+   archive
+      source.csv
+   scripts
+      README.md
+      py.py
+      requirements.txt - optional
+   README.md
+   datapackage.json
+   Makefile
+```
+
+In summary, you should fit under `data` the final CSV that you will use for your data package and the source data into the 
+`archive` folder.
+If you need to perform any script to clean and wrangle any bit of the dataset, you have to post it under `scripts`, preferably 
+with the name `process.py`, but this is not a convention.
+The `dir/README.md` should contain information about the package, source and licenses (if it applies). On the other hand, 
+`scripts/README.md` should talk about the script and any particular information about it. 
+
+*Note for Python users:* Do not forget to create the `requirements.txt` if you use any special Python package. 
+
+As for the Makefile, which is the easiest part, the common Makefile structure is as follows:
+
+```
+version="0.1.0"
+
+DATADIR=data
+SCRIPTDIR=scripts
+
+all: data
+
+data:
+      python $(SCRIPTDIR)/process.py
+
+      clean:
+            rm -f $(DATADIR)/*
+
+            .PHONY: all data clean
+```
+
+## The ugly job of quality assurance
```

Comment:
Let's keep these titles simpler and with less commentary ;-)

e.g. this could just be:

"Quality Assurance"